lib/suites.js

```diff
@@ -130,6 +136,16 @@ function _createEcdsaSd2023Suite({signer, options} = {}) {
   return diProof;
 }
 
+function _createEcdsaXi2023Suite({signer, options} = {}) {
+  // remove milliseconds precision
+  const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
```

Comment:
There's a helper function that was added for this recently to avoid repeating it in every one of these factory functions. You can take a look at the patterns used above to reuse it here.